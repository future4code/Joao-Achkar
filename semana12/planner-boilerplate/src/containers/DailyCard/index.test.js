import React from 'react'
import { DayCard, CardHeader, WeekDay } from './index'
import renderer from 'react-test-renderer'


test ("DayCard div test", () => {
    //preparação
    const newSnapShop = renderer.create(<DayCard/>).toJSON();
    //verificação 
    expect(newSnapShop).toMatchSnapshot();
})
test ("CardHeader div test", () => {
    //preparação
    const newSnapShop = renderer.create(<CardHeader/>).toJSON();
    //verificação 
    expect(newSnapShop).toMatchSnapshot();
})
test ("CardHeader div test", () => {
    //preparação
    const newSnapShop = renderer.create(<WeekDay/>).toJSON();
    //verificação 
    expect(newSnapShop).toMatchSnapshot();
})