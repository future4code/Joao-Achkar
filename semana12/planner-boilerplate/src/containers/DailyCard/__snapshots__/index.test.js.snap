// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader div test 1`] = `
<div
  className="sc-bwzfXH eJUgdU"
/>
`;

exports[`CardHeader div test 2`] = `
<h3
  className="sc-htpNat idDyKC"
/>
`;

exports[`DayCard div test 1`] = `
<div
  className="sc-bdVaJa gEDBe"
/>
`;
